Add support for using whisper models from Huggingface by specifying the model id. (#334)
guillaumekln/faster-whisper: README.md

# model = WhisperModel(model_size, device="cuda", compute_type="int8_float16")
# or run on CPU with INT8
# model = WhisperModel(model_size, device="cpu", compute_type="int8")
# or use available models (https://huggingface.co/models?search=faster-whisper) from Huggingface

> # model = WhisperModel('zh-plus/faster-whisper-large-v2-japanese-5k-steps')

Reviewer: I think this comment is not useful here. It's better to keep everything in the conversion section. Can you remove these lines?

Author: Done.

Models can also be converted from the code. See the [conversion API](https://opennmt.net/CTranslate2/python/ctranslate2.converters.TransformersConverter.html).

> ### Use converted model

Reviewer: ```suggestion
### Load a converted model
```

> 1. Directly use model from local directory:

Reviewer: ```suggestion
1. Directly load the model from a local directory:
```

```python
model = faster_whisper.WhisperModel('whisper-large-v2-ct2')
```

> 2. [Upload your model to Huggingface](https://huggingface.co/docs/transformers/model_sharing#upload-with-the-web-interface) and use online:

Reviewer: ```suggestion
2. [Upload your model to the Hugging Face Hub](https://huggingface.co/docs/transformers/model_sharing#upload-with-the-web-interface) and load it from its name:
```
